lib/server.go

```diff
@@ -242,7 +244,7 @@ func (srv *Server) GetMiner() *DeSoMiner {
 	return srv.miner
 }
 
-func (srv *Server) BroadcastTransaction(txn *MsgDeSoTxn) ([]*MempoolTx, error) {
+func (srv *Server) BroadcastTransaction(txn *MsgDeSoTxn) ([]*MsgDeSoTxn, error) {
```

Comment:
Just a note for myself - changing the return value here is perfectly fine since we don't actually use the first return param in backend or core right now.